docs/pages/core-concepts.mdx

```diff
@@ -44,23 +45,50 @@ infrastructure from the public internet without the need for a VPN.
 It establishes reverse tunnels to the **Teleport Auth Service** and **Teleport
 Services**, which can run in private networks. This means that, in the Proxy
 Service's minimal configuration, you can expose only port `443` to the internet
-and run the rest of your infrastructure in private networks.
+and run the rest of your infrastructure in private networks. You can also configure clients 
+to bypass Proxy Service instances and connect to resources with Teleport-issued certificates directly. 
 
-You can also configure clients to bypass Proxy Service instances and connect to
-resources with Teleport-issued certificates directly.
+As a key component of the Teleport control plane, the Proxy Service handles secure routing and session recording 
+for all incoming connections. For Teleport Cloud users, the Proxy Service is fully managed and hosted by Teleport.
 
-Read our guide to [how the Teleport Proxy Service
-works](reference/architecture/proxy.mdx).
+Read our guide to [how the Teleport Proxy Service works](reference/architecture/proxy.mdx).
 
 ## Teleport services
```

Comment:
I think it would make sense to merge this H2 with the Teleport Agent services H2, since otherwise it raises the question of what a service is vs. an agent service.